include/ebpf_helpers.h

```diff
@@ -6,6 +6,8 @@
 
 #include "ebpf_structs.h"
 
+#define EBPF_MAX_TAIL_CALL_DEPTH 32
```

Comment:
```suggestion
#define MAX_TAIL_CALL_CNT 32
```
use standard define name per bpf_tail_call man page